src/helm/proxy/clients/together_client.py

```diff
@@ -15,14 +15,19 @@
 
 
 _ASYNC_MODELS: Set[str] = {
+    "alpaca-7b",
```

Comment:
Document that these are the 'model aliases' rather than the Together model names?  I'm worried that there's going to be confusion if we have two namespaces that are very similar and overlap.

Reply:
This is documented in the docstring (and yes this is indeed confusing).